wiki/Gameplay/Slider_break/de.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - sliderbreak
+  - slider-break
+  - slider break
+  - combo break
+  - combobreak
+  - combo-break
+  - SB
+---
+
+# Sliderbreak
+
+**Sliderbreak** (manchmal abgekürzt als *SB*) ist ein Begriff, der verwendet wird, um das Unterbrechen einer Combo durch einen teilweise vollendeten [Slider](/wiki/Hit_object/Slider) zu beschreiben. Sliderbreaks können auf drei verschiedene Arten auftreten: den Kopf eines Sliders zu verfehlen, während der Körper noch ganz oder teilweise fertiggestellt ist, einen Slider zu irgendeinem Zeitpunkt seiner Dauer loszulassen oder nicht alle Ticks des Sliders zu sammeln. Alle drei Arten werden zusammen als "Sliderbreak" bezeichnet und beenden die Combo des Spielers. Ein Sliderbreak während eines Durchgangs führt zu einem "100er" und unterbricht die Combo, zählt aber nicht als Miss.
```

Comment:
```suggestion
**Sliderbreak** (manchmal abgekürzt als *SB*) ist ein Begriff, der verwendet wird, um das Unterbrechen einer Combo durch einen teilweise vollendeten [Slider](/wiki/Hit_object/Slider) zu beschreiben. Sliderbreaks können auf drei verschiedene Arten auftreten: den Kopf eines Sliders zu verfehlen, während der Körper noch ganz oder teilweise fertiggestellt ist, einen Slider zu irgendeinem Zeitpunkt seiner Dauer loszulassen oder nicht alle Ticks des Sliders zu sammeln. Alle drei Arten werden zusammen als "Sliderbreak" bezeichnet und beenden die Combo des Spielers. Ein Sliderbreak während eines Durchlaufs führt zu einem "100er" und unterbricht die Combo, zählt aber nicht als Miss.
```

Both is fine. `Durchlauf` sounds a little bit more common in this context in my opinion.